File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/database/room/RoomActivity.java
package com.jdroid.android.sample.database.room;

import com.jdroid.android.activity.FragmentContainerActivity;
import com.jdroid.android.sample.ui.analytics.AnalyticsFragment;

import androidx.fragment.app.Fragment;

public class RoomActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return RoomFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/database/room/RoomActivity.kt
package com.jdroid.android.sample.database.room

import androidx.fragment.app.Fragment
import com.jdroid.android.activity.FragmentContainerActivity

class RoomActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return RoomFragment::class.java
    }
}
